### PR TITLE
Upgrade the global operand from v1alpha3 to v1alpha4

### PR DESCRIPTION
**Pull request by yanmxa**
ref: https://issues.redhat.com/browse/ACM-6933
ref: https://issues.redhat.com/browse/ACM-6780

### CODE REVIEW

**Review by clyang82 — CHANGES_REQUESTED** [review]

File: operator/bundle/manifests/multicluster-global-hub-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -2,34 +2,12 @@ apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
   annotations:
-    alm-examples: |-
-      [
```
[the remark this answers is not in the document]
Comment (yanmxa) [reply operator/bundle/manifests/multicluster-global-hub-operator.clusterserviceversion.yaml:6]: Done.

**Review by clyang82 — APPROVED** [review]

**Pull request merged**